docs/UserGuide.md

```diff
@@ -43,144 +42,260 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 
 ### 2.1. Viewing help : `help`
 
-Format: `help`
+Format: `help [COMMAND]`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> * When there is no command specified, `help` will print help messages of all commands.
+> * Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
```

Comment:
2.2 add
- Since taskname is always required, would it be simpler for user if the "n/" is not needed?

- The format of duedate is not clear enough. e.g. only name of weekdays? how about date like 11/3/17? or common time specifier "tomorrow", "next Monday"etc?